pkg/operation/botanist/secrets.go

```diff
@@ -883,7 +945,10 @@ func (b *Botanist) generateShootSecrets(existingSecretsMap map[string]*corev1.Se
 // SyncShootCredentialsToGarden copies the kubeconfig generated for the user as well as the SSH keypair to
 // the project namespace in the Garden cluster.
 func (b *Botanist) SyncShootCredentialsToGarden() error {
-	for key, value := range map[string]string{"kubeconfig": "kubecfg", gardencorev1alpha1.SecretNameSSHKeyPair: gardencorev1alpha1.SecretNameSSHKeyPair} {
+	projectSecrets := map[string]string{"kubeconfig": "kubecfg", gardencorev1alpha1.SecretNameSSHKeyPair: gardencorev1alpha1.SecretNameSSHKeyPair}
+	projectSecrets["logging-credentials"] = "logging-ingress-credentials-users"
```

Comment:
Please call it only `logging,` not `logging-credentials` (see 2d3f929213150ab88d25de684eb4573f0c7107c9)